src/main/kotlin/pt/up/fe/ni/website/backend/model/Project.kt

```diff
@@ -17,6 +17,9 @@ class Project(
     @field:Size(min = Constants.Description.minSize, max = Constants.Description.maxSize)
     var description: String,
 
+    @JsonProperty(required = false)
+    var isArchived: Boolean = false,
```

Comment:
Two questions:
- Why is this not required? The user doesn't need to send it but we always have it when converting from our Dto
- Is there a reason why we have the default value here as well as in the Dto? Is the Dto not enough?

Reply:
You're right, the default value in the Dto is enough.

Reply:
Actually, when i remove the default value in the project class, the tests asks me to give one. 

<img width="573" alt="image" src="https://user-images.githubusercontent.com/87575749/203281607-8151369b-3973-4477-8f72-fdb586aef392.png">

Reply:
Have you tested using `val isArchived: Boolean? = false,` in ProjectDto?
I'm suggesting this because I think it makes sense to mark isArchived as required, however, I don't know how that solution would behave if a null value were to be passed on isArchived.

Reply:
Another option would be to leave it as it is and use @JsonProperty(required = false) on the ProjectDto just so it is clear to anybody looking at the code that it isn't a required argument and to put @JsonProperty(required = true) on the project itself

Reply:
Why would isArchived be required on the project itself? I think it should not be required. When a project is created, it should not be archived by default.

Reply:
The change in ProjectDto actually doesn't change any behaviour. `Boolean` is treated as a Java primitive and it always has a default value of `false` (even if you don't write it explicitly). I would personally leave the `= false` for readability.

As for the second discussion, @coutinho21 and @limwa , the `JsonProperty(required = true)` annotation doesn't mean that the `isArchived` field should be true by default. It means that, when converting a JSON object to a `Project`. Since we have a default value, I believe it should NOT be required. However, since that's the default behaviour, you don't even need to put the annotation